vnext/Microsoft.ReactNative/Fabric/FabricUIManagerModule.cpp

```diff
@@ -113,8 +113,10 @@ class AsyncEventBeat final : public facebook::react::EventBeat {
 std::shared_ptr<facebook::react::ComponentDescriptorProviderRegistry const> sharedProviderRegistry() {
   static auto providerRegistry = []() -> std::shared_ptr<facebook::react::ComponentDescriptorProviderRegistry> {
     auto providerRegistry = std::make_shared<facebook::react::ComponentDescriptorProviderRegistry>();
+    /*
```

Comment:
Did you mean to leave this in?

Reply:
Yeah, when someone implements ActivityIndicator they will just uncomment this code.  Hopefully it doesn't stay commented out for long.